src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -285,50 +285,6 @@ public JvbConference(AbstractGatewaySession gatewaySession, CallContext ctx)
             .getString(FOCUSE_RESOURCE_PROP, "focus");
     }
 
-    /**
-     * Includes info about given <tt>peer</tt> media SSRCs in MUC presence.
-     * @param peer the <tt>CallPeer</tt> whose media SSRCs will be advertised.
-     */
-    private void advertisePeerSSRCs(CallPeer peer)
```

Comment:
Is this just unnecessary since nothing reads SSRCs from precense?

Reply:
Yep, I searched around the namespace and didn't found anything ...